packages/angular_devkit/build_angular/src/webpack/plugins/hmr/hmr-accept.ts

```diff
@@ -27,6 +27,12 @@ export default function (mod: any): void {
     return;
   }
 
+  if (!isDevMode()) {
+    console.error(`[NG HMR] Cannot use HMR when Angular is running in production mode. Don't call enableProdMode() to enable development mode.`);
```

Comment:
```suggestion
    console.error(`[NG HMR] Cannot use HMR when Angular is running in production mode. To prevent production mode, do not call 'enableProdMode()'.`);
```